local-testing-docs/README.md

```diff
@@ -0,0 +1,315 @@
+# Snowpark Local Testing
+
+Hello and welcome to the Private Preview of the local testing framework for Snowpark. In this PrPr, you will be able to create DataFrames without a connection to Snowflake. This repository serves as a sample project, you can use this as a reference as you're getting started.
+
+For issues or questions, contact jason.freeberg@snowflake.com
+
+## Contents
+
+* [Quickstart](#quickstart)
+  + [Env setup](#env-setup)
+  + [Basic usage](#basic-usage)
+* [Overview](#overview)
+* [General Usage Documentation](#general-usage-documentation)
+  + [Installation](#installation)
+    - [Install from specific commit](#install-from-specific-commit)
+  + [Usage](#usage)
+  + [Usage with PyTest](#usage-with-pytest)
+  + [Usage with Behave](#usage-with-behave)
+* [Patching Built-In Functions](#patching-built-in-functions)
+* [SQL and Table Operations](#sql-and-table-operations)
+  + [Example](#example)
+* [Private Preview Notes](#private-preview-notes)
+* [Supported APIs](#supported-apis)
+  + [Session](#session)
+  + [DataFrame](#dataframe)
+  + [Scalar Functions](#scalar-functions)
+* [Limitations](#limitations)
+
+## Quickstart
+
+### Env setup
+
+1. Clone or download this repo if you haven't already
+1. Switch to this directory: `cd local-testing-docs`
+1. Create and activate the conda environment
+
+    ```bash
+    conda env create --file environment.yml
+    conda activate snowpark-local-test
+    ```
+
+### Basic usage
+
+1. Create a file, `quickstart.py`
+1. Add the following imports:
```

Comment:
The numbers in this section seems broken

Reply:
When it's rendered in Markdown the numbers are ordered correctly

Reply:
Interesting lol